src/AWS.Deploy.Orchestration/DeploymentBundleHandler.cs

```diff
@@ -362,10 +361,14 @@ private async Task PushDockerImage(string targetTagName)
         /// <returns>A dictionary that represents the environment varibales from the container</returns>
         public async Task<Dictionary<string, string>> InspectDockerImageEnvironmentVariables(Recommendation recommendation, string sourceTag)
         {
-            var dockerInspectCommand = "docker inspect --format \"{{ index (index .Config.Env) }}\" " + sourceTag;
+            var commandName = systemCapabilityEvaluator.GetInstalledContainerAppInfo()?.AppName?.ToLower();
+            if (string.IsNullOrEmpty(commandName))
+                throw new DockerBuildFailedException(DeployToolErrorCode.DockerBuildFailed, "No container app (Docker or Podman) is currently installed/running on your system.", -1);
+
+            var dockerInspectCommand = $"{commandName} inspect --format \"{{{{ index (index .Config.Env) }}}}\" " + sourceTag;
             if (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
-                dockerInspectCommand = "docker inspect --format '{{ index (index .Config.Env) }}' " + sourceTag;
-            var result = await _commandLineWrapper.TryRunWithResult(dockerInspectCommand, streamOutputToInteractiveService: false);
+                dockerInspectCommand = $"{commandName} inspect --format '{{{{ index (index .Config.Env) }}}}' " + sourceTag;
```

Comment:
this can technically be a docker or podman command, the variable `dockerInspectCommand` may not be accurate. can we just call it containerInspectCommand or something generic?

Reply:
same in other places too

Reply:
maybe deploymentInspectCommand since we are in deployment bundler class

Reply:
Updated the naming